Sources/SVGRenderer/SVGRenderer.swift

```diff
@@ -189,10 +189,16 @@ public class SVGRenderer: Renderer{
                               strokeWidth thickness: Float,
                               strokeColor: Color,
                               isDashed: Bool) {
-        guard !p.isEmpty else { return }
-        for i in 0..<p.count-1 {
-            drawLine(startPoint: p[i], endPoint: p[i+1], strokeWidth: thickness, strokeColor: strokeColor, isDashed: isDashed)
-        }
+        guard p.count > 1 else { return }
+        
+        let pointsString = p.map { point in
```

Comment:
`p.map` creates an Array, which we then immediately throw away after joining the elements together in to a String. It's _possible_ that the optimiser could remove the temporary, but not guaranteed or even predictable. `p.lazy.map` would always avoid the extra allocation, however.